core/modules/widgets/text.js

```diff
@@ -31,7 +31,9 @@ TextNodeWidget.prototype.render = function(parent,nextSibling) {
 	this.computeAttributes();
 	this.execute();
 	var text = this.getAttribute("text",this.parseTreeNode.text || "");
+	var length = this.getAttribute("length");
```

Comment:
what happens, if there is no length parameter? There seems to be no default value.